src/app/src/components/FacilitiesMapErrorMessage.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+
+const mapErrorMessageStyles = Object.freeze({
+    errorMessageContainerStyles: Object.freeze({
+        height: '100%',
+        width: '100%',
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+    }),
+});
+
+export default function FacilitiesMapErrorMessage() {
+    return (
+        <div style={mapErrorMessageStyles.errorMessageContainerStyles}>
+            <Typography variant="title">
+                An error prevented loading the map!
```

Comment:
Personal preference to not exclaim the error report.

Reply:
:+1: Changed in 9e27d85620b6f99f207d26bbd65e5523951b7ba9